modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/cell_renderers/DefaultRenderer.tsx

```diff
@@ -12,10 +12,12 @@ import React from 'react';
 import TooltipTextRenderer from './TooltipTextRenderer';
 
 interface DefaultRendererOptions {
+	fieldName?: string;
 	truncate?: boolean;
 }
 
 type DefaultRendererValue =
+	| any
```

Comment:
Try to use `LocalizedValue` from [liferay.d.ts](https://github.com/liferay-frontend/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/liferay.d.ts#L48) but don't understand why TS complains